src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,20 @@
 package mate.academy
 
+import mate.academy.service.FigureSupplier
+
+const val ARRAY_SIZE = 6
+
 fun main() {
-    // draw all figures here
+    val array = Array(ARRAY_SIZE) { FigureSupplier().getDefaultFigure() }
```

Comment:
Please check initialization logic.
Possible issue flow:
-initialization with random figures
-reinitialization with random figures on the next steps
-functionality duplication as result